hermes/probe/create/create_file.go

```diff
@@ -119,12 +133,28 @@ func CreateFile(ctx context.Context, target *probe.Target, fileID int32, fileSiz
 		return fmt.Errorf("CreateFile(id: %d).%v: could not create file %s: %w", fileID, status, fileName, err)
 	}
 	if err := wc.Close(); err != nil {
+		target.LatencyMetrics.APICallLatency[m.APICreateFile][m.WriterCloseFailed].Metric("hermes_api_latency_s").AddFloat64(time.Now().Sub(start).Seconds())
```

Comment:
`"hermes_api_latency_seconds"`, here & elsewhere

Reply:
"hermes_api_latency_s" is defined in metrics.go I did as per @leczb suggestion add a constant hermesAPILatencySeconds="hermes_api_latency_seconds"

Reply:
If we're defining latency names in metrics.go, can we just define the metric names as exported constants there, rather than having them in several places (and things breaking if they stop matching)